modules/openapi-generator/src/main/resources/C-libcurl/any_type-header.mustache

```diff
@@ -0,0 +1,5 @@
+/*
```

Comment:
If you need add header or source files, you can add a "PreTarget.cmake", e.g. https://github.com/kubernetes-client/c/blob/master/kubernetes/PreTarget.cmake